tests/v2/test_peer_store_extended.nim

```diff
@@ -0,0 +1,268 @@
+{.used.}
+
+import
+  std/[options,sequtils],
+  libp2p/crypto/crypto,
+  libp2p/peerstore,
+  libp2p/multiaddress,
+  testutils/unittests
+import
+  ../../waku/v2/node/peer_manager/peer_manager,
+  ../../waku/v2/node/peer_manager/waku_peer_store,
+  ../../waku/v2/node/waku_node,
+  ../test_helpers,
+  ./testlib/testutils
+
+
+suite "Extended nim-libp2p Peer Store":
+  # Valid peerId missing the last digit. Useful for creating new peerIds
+  # basePeerId & "1"
+  # basePeerId & "2"
+  let basePeerId = "QmeuZJbXrszW2jdT7GdduSjQskPU3S7vvGWKtKgDfkDvW"
+  
+  # Given
```

Comment:
_Given_ is for the test cases. It is optional for the test cases like "setup" and "cleanup". On the other hand, "when" and "then" are mandatory. 

So, "Given" is not intended to be added globally. For that, you have _Setup_. And it is usually for doing things like what you are doing here: _configuring shared fixtures_.